mne/io/egi/tests/test_egi.py

```diff
@@ -136,24 +136,29 @@ def test_io_egi_mff():
                            test_scaling=False,  # XXX probably some bug
                            )
     assert raw.info['sfreq'] == 1000.
-    # The ref here is redundant, but we don't currently have a way in
-    # DigMontage to mark that a given channel is actually the ref so...
-    assert len(raw.info['dig']) == 133  # 129 eeg + 1 ref + 3 cardinal points
-    assert raw.info['dig'][0]['ident'] == 1  # EEG channel E1
-    assert raw.info['dig'][3]['ident'] == 0  # Reference channel
-    assert raw.info['dig'][-1]['ident'] == 129  # Reference channel
-    ref_loc = raw.info['dig'][3]['r']
+    assert len(raw.info['dig']) == 132  # 3 cardinal points + 129 eeg (one is ref)
+    assert raw.info['dig'][0]['ident'] == FIFF.FIFFV_POINT_LPA
+    assert raw.info['dig'][0]['kind'] == FIFF.FIFFV_POINT_CARDINAL
+    assert raw.info['dig'][3]['kind'] == FIFF.FIFFV_POINT_EEG
+    assert raw.info['dig'][-1]['ident'] == 129
+    assert raw.info['custom_ref_applied'] == FIFF.FIFFV_MNE_CUSTOM_REF_ON
+    ref_loc = raw.info['dig'][-1]['r']
     eeg_picks = pick_types(raw.info, eeg=True)
     assert len(eeg_picks) == 129
+    # ref channel doesnt store its own loc as ref location
+    # so don't test it
+    ref_pick = pick_channels(raw.info['ch_names'], ['VREF'])
+    eeg_picks = np.setdiff1d(eeg_picks, ref_pick)
     for i in eeg_picks:
         loc = raw.info['chs'][i]['loc']
         assert loc[:3].any(), loc[:3]
         assert_array_equal(loc[3:6], ref_loc, err_msg=f'{i}')
     assert raw.info['device_info']['type'] == 'HydroCel GSN 128 1.0'
 
     assert 'eeg' in raw
+    # test our custom channel naming logic functionality
     eeg_chan = [c for c in raw.ch_names if 'EEG' in c]
-    assert len(eeg_chan) == 129
+    assert len(eeg_chan) == 128  # VREF will not match in comprehension
```

Comment:
rather than hard-coding a bunch of numbers throughout this test, it might be better to define a few convenience variables at the start of the test, that we can test against.  Sometimes we call these "want variables", i.e., what we want/expect the tested functions to return.  e.g., `want_nchans = 132` and then later `assert nchans == want_nchans`.  Here, I would probably drop the "want" to be more concise, and do something like this maybe?
```py
n_ref = 1
n_eeg = 128
n_card = 3
```
...and then the assert lines become something like `assert len(eeg_picks) == n_eeg + n_ref` or, here, `assert len(eeg_chan) == n_eeg  # VREF will not match`